.gitpod.yml

```diff
@@ -0,0 +1,27 @@
+image:
+  file: .gitpod.Dockerfile
+
+tasks:
+  - name: Create mamba environment and start the dev server
+    before: |
+      mamba env create -n mkdocs --file environment.yml
+    command: |
+      conda activate mkdocs
+      mkdocs serve
```

Comment:
oooh is this fancy like jupyter notebooks and allows you to serve the website from gitpod but you can view it in your browser?

Reply:
yup exactly.

Reply:
very cool!